connectivity/check/policy.go

```diff
@@ -26,6 +26,7 @@ const (
 	ExitAnyError    ExitCode = -1
 	ExitInvalidCode ExitCode = -2
 
+	ExistInvalidHost  ExitCode = 6
```

Comment:
Oh, and is this change intentional? I don't see this being used anywhere in the PR.

Reply:
Is the `Exist` prefix here intended, maybe it should be `Exit`, i.e., `ExitInvalidHost`?